src/libnnstreamer-edge/nnstreamer-edge-custom-impl.h

```diff
@@ -0,0 +1,85 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/**
+ * Copyright (C) 2024 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * @file   nnstreamer-edge-custom-impl.h
+ * @date   21 Aug 2024
+ * @brief  Internal interface to support communication using custom library.
+ * @see    https://github.com/nnstreamer/nnstreamer
+ * @author Gichan Jang <gichan2.jang@samsung.com>
+ * @bug    No known bugs except for NYI items
+ */
+
+#ifndef __NNSTREAMER_EDGE_CUSTOM_IMPL_H__
+#define __NNSTREAMER_EDGE_CUSTOM_IMPL_H__
+
+#include "nnstreamer-edge-custom.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+
+/**
+ * @brief Data structure for edge custom connection.
+ */
+typedef struct
+{
+  void *dl_handle;
+  void *instance;
```

Comment:
If `instance` is defined as `nns_edge_custom_s *` type instead of `void *` type, it seems that repetitive casting can be reduced.
```suggestion
  nns_edge_custom_s *instance;
```

Reply:
I updated param type and function ptr to load library. plz review the changes again.